README.md

```diff
@@ -1,24 +1,51 @@
-# README
+[](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# BlogProject
 
-Things you may want to cover:
+> Ruby on Rails based project
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby
+- RSpec
+- Postgres Sql
 
-* Database creation
+## Getting Started
 
-* Database initialization
+To get a local copy follow these simple example steps.  
 
-* How to run the test suite
+1.- Open the terminal window and clone the repository using this command:  
+`git clone https://github.com/AlejandroNeira98/BlogProject.git` 
 
-* Services (job queues, cache servers, search engines, etc.)
+2.- Change the directory to the school-library directory  
+`cd BlogProject`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`
+4.- In PostgreSQL create a database, remember the name, the owner and the password
+5.- Go to config/database.yml under development specify the database name, owner(username), and password
+6.- To migrate from rails to your database run 'rails db:migrate'
```

Comment:
- [ ] [Optional] Please make the user run `rails db:create` first before he run the migration files.